src/site/xdoc/google_style.xml

```diff
@@ -1975,11 +1975,9 @@
                         src="images/ban_red.png"
                         alt="" />
                   </span>
-                  That validation could not be checked by Checkstyle. It's
-                  need to
-                  take a look as parent class. But Checkstyle have no way to
-                  open or
-                  look at another Class file.
+                  Validation of this requirement can not be done by Checkstyle.
+                  It needs information from the parent class, defined in another Java file,
+                  but Checkstyle has no way to read another Java file.
```

Comment:
Same as above. Messages should be exactly the same, with a link to our limitations.